cmd/mattermost/commands/command.go

```diff
@@ -67,3 +78,38 @@ func moveCommandCmdF(command *cobra.Command, args []string) error {
 func moveCommand(a *app.App, team *model.Team, command *model.Command) *model.AppError {
 	return a.MoveCommand(team, command)
 }
+
+func deleteCommandCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	if len(args) < 1 {
```

Comment:
We can use here the "cobra.MinimumNArgs(1)` instead of this like you can see here: https://github.com/mattermost/mattermost-server/pull/9555/files#diff-f4e304fd7dca99ff88df6e390dbbd346R117

Reply:
Done.